scratch.md

```
@@ -4540,3 +4540,39 @@ chrome://help - version
+ 		- [ ] finish	
+
+- [ ] fix config double click
+- [ ] config
+  - [ ] vmmapping
+  - [ ] my after
+  - [ ]  interface
+  - [ ] doc designer
+    - [ ] dd body quote table xsl
+
+- [ ] quick links
+ - [ ] demo config - config question
```

markdownlint-cli2 (0.17.2)

Inconsistent indentation for list items at the same level
Expected: 0; Actual: 1

(MD005, list-indent)